src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      'x': 0,
+      'y': 0,
+    },
+
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(moveBy = 1) {
+      if (moveBy < 0) {
+        return this;
+      }
+      this.coords.y += moveBy;
+
+      return this;
```

Comment:
Follow DRY - don't repeat yourself, you can invert your condition so it looks like this, fix it everywhere
```suggestion
      if (moveBy > 0) {
         this.coords.y += moveBy;
      }

      return this;
```